### PR TITLE
Fixed patient to be only considered duplicate, if name, phone, email and address are all the same

### PR DESCRIPTION
**Pull request by Ramanathan0908**

### CODE REVIEW

File: src/main/java/seedu/uninurse/model/person/Patient.java [hunk 1]
```diff
@@ -132,7 +132,9 @@ public boolean isSamePerson(Patient otherPerson) {
         }
 
         return otherPerson != null
-                && otherPerson.getName().equals(getName());
+                && otherPerson.getName().equals(getName())
```
Comment (BlopApple) [inline src/main/java/seedu/uninurse/model/person/Patient.java:135]: I think you missed out email here.

**Review by BlopApple — CHANGES_REQUESTED** [review]
LGTM except for a minor mistake.

**Comment by izzahaj** [comment]
You CI checks are failing due to an unresolved compile error from `CommandResult` that somehow got merged. I have resolved it so it would be great to pull and update your branch

**Review by BlopApple — APPROVED** [review]
LGTM

**Pull request merged**